pipelines/template-build/template-build.yaml

```diff
@@ -159,6 +159,8 @@ spec:
         value: registry.access.redhat.com
       - name: IMAGE_REPOSITORY
         value: $(tasks.sanity-inspect-image.results.BASE_IMAGE_REPOSITORY)
+      - name: BASE_IMAGES_DIGESTS
+        value: $(tasks.build-container.results.BASE_IMAGES_DIGESTS)
```

Comment:
By using a result from the build-container task, this task (deprecated-base-image-check) will be executed after the build-container task. I think that's ok, and actually the correct behavior we want, but the `runAfter` attribute a couple of lines above is a bit confusing. Maybe consider removing it altogether, and just let Tekton decide when the task should run based on the results it depends on?

Reply:
Sure. will do.

Reply:
removed